tasks.py

```diff
@@ -110,13 +106,13 @@ def install_local(ctx, package):
     that is a cross between all local packages requested. It may
     not be stable.
     """
-    if package:
-        packages = _get_package_paths()
-        for pkg in package:
-            # activate meta context
-            with ctx.prefix(ACTIVATE):
-                # uninstall package
-                pip(ctx, f"uninstall {pkg} -y")
-                # cd to selected package and run poetry install
-                with ctx.cd(packages[pkg]):
-                    poetry(ctx, "install")
+    if not packages:
```

Comment:
Is called _early exit_ and draws the attention that if there are no `packages`, then just quit.
Bonus: You unindent the code block below one level. (since isn't guarded by the `if` anymore)

If you don't like it, you can drop it :).